modules/uncertainty/R/ensemble.R

```diff
@@ -215,22 +215,24 @@ write.ensemble.configs <- function(defaults, ensemble.samples, settings, model,
   
   
   # Check if there are NO inputs
-  if (is.null(defaults$inputs)) {
-    PEcAn.logger::logger.severe("No inputs provided - cannot configure ensemble")
+ 
+for (input_tag in names(settings$run$inputs)) {
+  input <- settings$run$inputs[[input_tag]]
+  input_paths <- input$path
+  
+  # Check for required paths
+  if (is.null(input_paths) || length(input_paths) == 0) {
+    PEcAn.logger::logger.error("Input '%s' has no paths specified", input_tag)
```

Comment:
Looks like you're trying to use sprintf style here, but logger.error isn't that smart -- it just pastes all its arguments together.
```suggestion
    PEcAn.logger::logger.error("Input", sQuote(input_tag), "has no paths specified")
```